src/lib.rs

```diff
@@ -50,7 +49,7 @@ pub enum ControlOutcome {
 
 /// The current meta-tracing state of a given location in the end-user's code.
 #[derive(Debug)]
-enum LocCount {
+enum LocPhase {
```

Comment:
So this is an abbreviation of "Location Phase"? Does a location have a phase? Or is the phase a property of the thread?

Reply:
Yes, yes, no (respectively) :)